server/aws-lsp-codewhisperer/src/shared/codeWhispererService.ts

```diff
@@ -543,20 +543,37 @@ export class CodeWhispererServiceToken extends CodeWhispererServiceBase {
 
     private mapCodeWhispererApiResponseToSuggestion(
         apiResponse: GenerateCompletionsResponse,
+        request: GenerateTokenSuggestionsRequest,
```

Comment:
It looks like you only need `request.predictionTypes`? Can we make the argument type `PredictionType[]` instead?

Reply:
yea makes sense to me, updated Tai.